Vtiger_CRM/src/test/java/contacts/DeleteContact.java

```diff
@@ -0,0 +1,5 @@
+package contacts;
+
+public class DeleteContact {

```

Comment:
set the path and launch the browser

Reply:
follow all the indentaions and coding standards

Reply:
yes all the comments have been implemented